File: official/docs/node/insurances/create_insurance.js
const Easypost = require('@easypost/api');
const api = new Easypost('EASYPOST_API_KEY');

const insurance = new api.Insurance({
    to_address: {id: "adr_..."},
    from_address: {id: "adr_..."},
    tracking_code: '9400110898825022579493',
    carrier: 'USPS',
    amount: '100.00',
    reference: 'insuranceRef1'
});
  
insurance.save().then(console.log);

File: official/docs/go/insurances/create_insurance.go
package example

import (
    "os"

    "github.com/EasyPost/easypost-go/v2"
)

func main() {
	apiKey := os.Getenv("EASYPOST_API_KEY")
    client := easypost.New(apiKey)

	toAddress, err := client.GetAddress("adr_...")
    fromAddress, err := client.GetAddress("adr_...")

    insurance, err := client.CreateInsurance(
        &easypost.Insurance{
            ToAddress:    toAddress,
            FromAddress:  fromAddress,
            Reference:    "InsuranceRef1",
            Carrier:      "USPS",
            TrackingCode: "9400110898825022579493",
            Amount:       "100.00",
        },
    )

	fmt.Println(insurance)
}
